zeebe/engine/src/test/java/io/camunda/zeebe/engine/util/ProcessingExporterTransistor.java

```diff
@@ -45,8 +45,7 @@ public void onRecovered(final ReadonlyStreamProcessorContext context) {
         () -> {
           final int partitionId = context.getPartitionId();
           typedEvent = new TypedRecordImpl(partitionId);
-          final var asyncLogStream = synchronousLogStream.getAsyncLogStream();
-          asyncLogStream.registerRecordAvailableListener(this::onNewEventCommitted);
+          synchronousLogStream.registerRecordAvailableListener(this::onNewEventCommitted);
```

Comment:
:wrench: Since we renamed the class too, might as well rename the field/variable.

```suggestion
          logStream.registerRecordAvailableListener(this::onNewEventCommitted);
```